tests/pipeline/test_ray_executor.py

```diff
@@ -112,3 +112,22 @@ def test_remote_wrapper_sets_name_based_on_task_info():
     def funcname(): pass
     wrapper = get_remote_wrapper(PipelineTask('taskname', funcname), None)
     assert wrapper.__name__ == 'taskname<funcname>'
+
+
+@pytest.mark.parametrize('batch_size', [None, 1, 10])
+def test_remote_wrapper_wrapped_func(raysession, batch_size):
+    def func(arg):
+        return arg
+    wrapper = get_remote_wrapper(PipelineTask('taskname', func, batch_size=batch_size), None)
+    df_in = pd.DataFrame({'col': range(20)})
+    res = wrapper(df_in)
+    assert len(res) == len(df_in)
```

Comment:
[pedantic] ``df`` is mutable by ``wrapper`` (at least it could be - without knowing more) so the length could change so would be better to comp prior to calling or assign to an attr ``length = 20``.

Reply:
ah yes, thanks. I will fix that and push considering this sticky